src/posthog-core.ts

```diff
@@ -1815,6 +1815,11 @@ export class PostHog {
         return !!this.sessionRecording?.started
     }
 
+    /** Checks the feature flags associated with this Survey to see if the survey can be rendered. */
+    captureException(error: Error, additionalProperties?: Properties): void {
+        this.exceptionObserver?.captureException(error, additionalProperties)
```

Comment:
hmmmm....

exception capture is ~10% of the bundle if it wasn't lazily loaded.

if someone doesn't have exception capture enabled posthog will still advertise this method as part of its contract, what should it do?

just log?

Reply:
> exception capture enabled

this is mostly about the auto-capture, right? people could still do `capture(thisError)` manually, if we allow that, it should always be part of the bundle.
Maybe we need a different config for auto-capture and manual capture, so people that don't use either, don't pay the bundle price.